pages/agent/v3/help/_env_dump.md

```diff
@@ -13,9 +13,8 @@ script.
 -->
 
 ### Usage
-
-`buildkite-agent env dump [options]`
-
+buildkite-agent env dump [options]
+``
```

Comment:
This looks like a mistake:

<img width="331" alt="Screenshot 2023-03-21 at 10 15 38 am" src="https://user-images.githubusercontent.com/119824349/226485820-4def3a55-e061-4c18-b180-fdc0ed103d67.png">